Jenkinsfile.prod

```diff
@@ -8,6 +8,13 @@ elifePipeline {
     stage 'Deploy to prod', {
         elifeDeploySlackNotification 'annotations', 'prod'
         elifeGitMoveToBranch commit, 'master'
+        elifeOnNode(
+            {
+                sh "docker tag elifesciences/annotations_cli:${commit} elifesciences/annotations_cli:latest && docker push elifesciences/annotations_cli:latest"
+                sh "docker tag elifesciences/annotations_fpm:${commit} elifesciences/annotations_fpm:latest && docker push elifesciences/annotations_fpm:latest"
```

Comment:
Is `latest` standard Docker terminology? Or could we use `master` to match the branch name?

Reply:
It is a convention derived by the default tag used by `docker pull`:
https://hub.docker.com/r/jenkins/jenkins/tags/
https://docs.docker.com/engine/reference/commandline/pull/#examples